packages/nvidia_nat_langchain/src/nat/plugins/langchain/agent/react_agent/agent.py

```
@@ -155,7 +155,7 @@ def _get_tool(self, tool_name: str):
             logger.error("%s Unable to find tool with the name %s\n%s", AGENT_LOG_PREFIX, tool_name, ex)
             raise
 
-    async def agent_node(self, state: ReActGraphState):
+    async def agent_node(self, state: ReActGraphState, config: RunnableConfig) -> ReActGraphState:
```

Ruff (0.15.0)

[warning] Unused method argument: `config`

(ARG002)